src/util/sharing_map.h

```diff
@@ -919,10 +919,11 @@ SHARING_MAPT(void)::add_item_if_not_shared(
       if(equalT()(leaf.get_key(), ip->get_key()))
       {
         delta_view.push_back({k, leaf.get_value(), ip->get_value()});
-        return;
       }
-
-      delta_view.push_back({k, leaf.get_value()});
+      else if(!only_common)
+      {
+        delta_view.push_back({k, leaf.get_value()});
+      }
```

Comment:
Wouldn't the same apply in line 909?

Reply:
Yes, I believe so - our comments crossed in the ether :slightly_smiling_face: - but I don't have a case that hits that line.

Reply:
I'd like the build to complete, and then I'll push that change too.